src/components/SettingsPage.vue

```diff
@@ -26,6 +26,7 @@
                   type="file"
                   class="custom-file-input"
                   id="customFile"
+                  v-on:change="profilePicture"
```

Comment:
Rename the function to `handleProfilePicture` and also add it in methods

Reply:
change "profilePicture" to "handleProfilePicture" ?

Reply:
"profilePicture" to "handleProfilePicture"?

Reply:
first one is a field, another is a function

Reply:
@vikas339 unaddressed comment